### ISSUE TITLE
Создание/редактирование предопределенных элементов

### ISSUE THREAD

**Issue by alekseineputin** [cpr1c/tools_ui_1c#310]

**Comment by LevkinSergey** [comment]
Есть готовые мысли, примеры как это лучше реализовать?

**Comment by alekseineputin** [comment]
> готовые мысли, примеры
Нет. Сегодня при использовании редактора реквизитов возникла необходимость проверить, является ли выбранный мной элемент предопределенным.

> как это лучше реализовать
Предлагаю сделать закладку "Стандартные реквизиты" и вывести их туда.

**Comment by LevkinSergey** [comment]
Реализовано в тестовой версии
